src/05-derivatives/02-common-data-types.md

```diff
@@ -0,0 +1,311 @@
+
+Discrete segmentations of brain tissue represent each tissue class with a unique
+integer label in a 3D volume. See "Anatomical Labels" for interpretation how
+integer values map to tissue classes.
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        func|anat|dwi/
+            <source_keywords>[_space-<space>]_dseg.nii.gz
+```
+
+Example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_space-orig_dseg.nii.gz
+            sub-001_space-orig_dseg.json
+```
+
+A segmentation could be a binary mask that functions as a discrete `label` for a
+single structure. In this case, the label key must be used to specify the
+corresponding structure. For example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_space-orig_label-GM_dseg.nii.gz
+```
+
+See "Anatomical labels" for reserved key values for `label`.
+
+### Probabilistic Segmentations
+
+Probabilistic segmentations of brain tissue represent a single tissue class with
+values ranging from 0 to 1 in individual 3D volumes or across multiple frames.
+Similarly to a discrete, binary segmentation, the label key can be used to
+specify the corresponding structure.
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        func|anat|dwi/
+            <source_keywords>[space-<space>][_label-<label>]_probseg.nii.gz
+```
+
+Example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_space-orig_label-BG_probseg.nii.gz
+            sub-001_space-orig_label-WM_probseg.nii.gz
+```
+
+See "Anatomical labels" for reserved key values for `label`.
+
+A 4D probabilistic segmentation, in which each frame corresponds to a different
+tissue class, must provide a label mapping in its JSON sidecar. For example:
+
+```Text
+pipeline/
+    sub-001/
+	    anat/
+		    sub-001_space-orig_probseg.nii.gz
+		    sub-001_space-orig_probseg.json
+```
+
+The JSON sidecar must include the label-map key that specifies a tissue label
+for each volume:
+
+```JSON
+{
+    "LabelMap": [
+        "BG",
+        "WM",
+        "GM"
+        ]
+}
+```
+
+Values of `label` need to map to Abbreviations defined in "Anatomical Labels".
+
+### Surface Parcellations
+
+Discrete parcellations (surface segmentations) of cortical structures should be
+stored as GIFTI or CIFTI file.
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        anat/
+            <source_keywords>[_hemi-{L|R}][_space-<space>]_dseg.{label.gii|dlabel.nii}
+```
+
+The REQUIRED extension for GIFTI parcellations is `.label.gii`. The hemi tag is
+REQUIRED for GIFTI files. For example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_hemi-L_dparc.label.gii
+            sub-001_hemi-R_dparc.label.gii
+```
+
+The REQUIRED extension for CIFTI parcellations is `.dlabel.nii`. For example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_dparc.dlabel.nii
+            sub-001_dparc.dlabel.nii
```

Comment:
```suggestion
            sub-001_dseg.dlabel.nii
```

Reply:
Yeah. I think there is a history of referring to atlases in nifti as segmentation and atlases on the surface as parcellations, so `_dseg` and `_dparc` suffixes appeared. but I agree that there is no reason for this.